examples/zkdetect/main.go

```diff
@@ -0,0 +1,53 @@
+package main
```

Comment:
i want to run this on the command line. my understanding of testing
examples is that go-test runs them but we don't get separate binaries that
we can execute manually

On Mon, Sep 14, 2015 at 9:12 AM, Tomás Senart <notifications@github.com>
wrote:

> In examples/zkdetect/main.go
> <https://github.com/mesos/mesos-go/pull/174#discussion_r39391156>:
>
> > @@ -0,0 +1,53 @@
> > +package main
>
> Can't we use testing examples instead?
> http://golang.org/pkg/testing/#hdr-Examples
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mesos/mesos-go/pull/174/files#r39391156>.
>